Sources/PrimerSDK/Classes/Data Models/PrimerSettings.swift

```diff
@@ -85,7 +85,7 @@ public class PrimerSettings: PrimerSettingsProtocol {
 //    internal(set) public var merchantCapabilities: [MerchantCapability]?
     internal(set) public var isInitialLoadingHidden: Bool
     internal(set) public var localeData: LocaleData
-    internal(set) public var is3DSEnabled: Bool
+    internal(set) public var is3DSOnVaultingEnabled: Bool
```

Comment:
why on vaulting?

Reply:
Because on checkout it's working dynamically, this won't affect anything.